PURL-SPECIFICATION.rst

```diff
@@ -225,17 +227,13 @@ Character encoding
 Permitted characters
 --------------------
 
-A canonical ``purl`` is an ASCII string composed of these characters:
+A canonical ``purl`` is composed of these characters ("Permitted Characters"):
```

Comment:
Do we really need `("Permitted Characters")` here? What about this instead:
```suggestion
A canonical ``purl`` is composed of these Permitted Characters:
```

Reply:
@pombredanne Done.  Are you suggesting we keep using "Characters" in this title rather than "Bytes" as @ppkarwasz suggested?

Reply:
Let's reference this as permitted ASCII characters.